sympy/series/tests/test_limits.py

```diff
@@ -139,7 +139,7 @@ def test_log():
     expr = tau**2*((tau - 1)*(tau + 1)*log(x + 1)/(tau**2 + 1)**2 + 1/((tau**2\
             + 1)*(x + 1)) - (-2*tau*atan(x/tau) + (tau**2/2 - 1/2)*log(tau**2\
             + x**2))/(tau**2 + 1)**2)
-    assert limit(expr, x, oo) == pi*tau**3/(tau**2 + 1)**2
+    assert limit(expr, x, oo) == N(pi)*tau**3/(tau**2 + 1)**2
```

Comment:
It seems that this is inexact because of 1/2 in expr. Maybe we could make that exact without essentially changing the test.

Reply:
I had thought so too but wasn't sure if the test should be changed, or the inexact 1/2 should intentionally have been preserved... I will change it to S.Half